### PR TITLE
Add note to writing points from a file

### PR DESCRIPTION
**Pull request by rkuchan**
Fixes https://github.com/influxdb/influxdb.com/issues/357

### CODE REVIEW

File: content/docs/v0.9/guides/writing_data.md [hunk 1]
```diff
@@ -54,6 +54,8 @@ Write the data in `cpu_data.txt` to the `mydb` database with:
 <br>
 `curl -i -XPOST 'http://localhost:8086/write?db=mydb' --data-binary @cpu_data.txt`
 
+> **Note:** If your data file is big enough, it may be necessary to split that file into several files in order to write your data in batches to InfluxDB. [By default](https://github.com/influxdb/influxdb/blob/master/etc/config.sample.toml#L97), InfluxDB does not complete write operations that take longer than five seconds.
```
Comment (gunnaraasen) [inline content/docs/v0.9/guides/writing_data.md]: Lets say, "has more than 5000 points" instead of "big enough".

Comment (gunnaraasen) [inline content/docs/v0.9/guides/writing_data.md]: I may be wrong, but I think InfluxDB will still attempt to write the points even if the HTTP request times out at 5 seconds. There's just no confirmation they were written. Also, the timeout is configurable.

**Comment by gunnaraasen** [comment]
Left a couple comments. Otherwise it looks good!

**Comment by jackzampolin** [comment]
:+1:

**Pull request merged**